File: Beecrowd/5-Matemcatica/3104-Resto 2.0-10/3104.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var l = input.split('\n').filter(line => line.trim() !== '');


let i = 0;
while(i<l.length){
    let A = l[i];
    i++;
    let B = parseInt(l[i]);
    i++
    let resto = 0;
    for(let c of A){
        resto = (resto * 10 + parseInt(c)) % B;
    }
    console.log(resto);
}

File: Beecrowd/5-Matemcatica/3104-Resto 2.0-10/3104.cpp
#include <iostream>
#include <string>
using namespace std;
int main() {
    string A;
    long long B, resto = 0;
    cin >> A >> B;
    for (char c : A) {
        resto = (resto * 10 + (c - '0')) % B;
    }
    cout << resto << endl;
    return 0;
}
